upp-delivery-provisioner/ansible/userdata/stateless_instance_user_data.yaml

```diff
@@ -103,6 +103,7 @@ coreos:
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/neo4j/read_write_url '${neo4j_write_url}'                                                >/dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/aggregate-concept-transformer/bucket '${aggregate_concept_bucket}'                       >/dev/null 2>&1 || true;"
           ExecStart=/bin/sh -c "etcdctl mk /ft/config/aggregate-concept-transformer/queue_url '${aggregate_concept_queue}'                     >/dev/null 2>&1 || true;"
+          ExecStart=/bin/sh -c "etcdctl mk /ft/config/aggregate-concept-transformer/dynamodb_table '${dynamo_table}'                           >/dev/null 2>&1 || true;"
```

Comment:
I think your `${dynamo_table}` variable name doesn't match what's in initial_stateless_user_data.yaml - in that file, it's `dynamodb_table`.